CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+## Contribution Guidelines
+
+**Note:** If these contribution guidelines are not followed your issue or PR might be closed, so
+please read these instructions carefully.
+
+_See also: [Flutter's code of conduct](https://flutter.dev/design-principles/#code-of-conduct)_
+
+## About
+
+Melos is a tool that optimizes the workflow around managing multi-package repositories with git and Pub.
+
+## Contribution types
+
+### Bug Report
+
+- If you find a bug, please first report it using [Github issues].
+  - First check if there is not already an issue for it; duplicated issues will be closed.
```

Comment:
Is this item meant to be indented?

Reply:
Yes, I thought this was a part of the above point.